src/domain/entities/TeamMember.ts

```diff
@@ -0,0 +1,24 @@
+/**
+ * Team member entity
+ */
+export interface TeamMember {
+  /**
+   * team member id
+   */
+  id: number;
+
+  /**
+   * team member noteId
+   */
+  noteId: number;
+
+  /**
+   * team member userid
+   */
+  userId: string;
+
+  /**
+   * team member role
```

Comment:
Provided docs are useless. Please, write docs explaining the reason of each property exists.